app/javascript/packs/document-capture.tsx

```diff
@@ -178,7 +178,7 @@ render(
                       isSelfieCaptureEnabled: getSelfieCaptureEnabled(),
                       isSelfieDesktopTestMode: String(docAuthSelfieDesktopTestMode) === 'true',
                       showHelpInitially: true,
-                      immediatelyBeginCapture: true,
+                      immediatelyBeginCapture: false,
```

Comment:
Is this the only place we had been passing `immediatelyBeginCapture` ? Can we remove it altogether?

Reply:
Yeah looks like we don't need it. Had to confirm with the team before i outright removed it. Will do.

Reply:
After testing we actually need `immediatelyBeginCapture` it might be in the acuant docs somewhere

Reply:
I don't see any [references to that term in Acuant's source code](https://github.com/search?q=repo%3AAcuant%2FJavascriptWebSDKV11%20immediatelyBeginCapture&type=code). What were the issues you were bumping into?